test/e2e/storage/external/external.go

```diff
@@ -242,6 +243,9 @@ func (d *driverDefinition) GetDynamicProvisionStorageClass(config *testsuites.Pe
 
 	sc, ok := items[0].(*storagev1.StorageClass)
 	gomega.Expect(ok).To(gomega.BeTrue(), "storage class from %s", d.StorageClass.FromFile)
+	// Ensure that we can load more than once as required for
+	// GetDynamicProvisionStorageClass by adding a random suffix.
+	sc.Name = names.SimpleNameGenerator.GenerateName(sc.Name)
```

Comment:
Generate name will directly append a 5 character random string to the end of the input. Could you add a `-` character between the original sc.Name and the random string for readability?

Reply:
Fixed and force-pushed.